### PR TITLE
Feature / TEC-3545 icons accent color

### PR DESCRIPTION
**Pull request by alliuca**
**Description:** Displays icons in the accent color set inside the customizer.

**Ticket:** https://moderntribe.atlassian.net/browse/TEC-3545

**Screencast:** http://p.tri.be/bCQnDz

### CODE REVIEW

File: src/Tribe/Customizer/Global_Elements.php [hunk 1]
```diff
@@ -138,6 +138,19 @@ public function get_css_template( $template ) {
 					background-color: ' . $accent_color_background . ';
 				}
 			';
+			
+			// Override svg icons color.
+			$template .= '
+				.tribe-common .tribe-common-c-svgicon {
+					color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:145]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 7e2242af3350820508e936e52d89744e</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 2]
```diff
@@ -138,6 +138,19 @@ public function get_css_template( $template ) {
...
+				}
+			';
+			
+			$template .= '
+				.tribe-common .tribe-events-virtual-virtual-event__icon-svg {
+					color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:151]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 01926ec495f9ee79ed3ab0ef23bdec6c</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 2 warning(s)

File: src/Tribe/Customizer/Global_Elements.php [hunk 3]
```diff
@@ -139,6 +139,19 @@ public function get_css_template( $template ) {
 				}
 			';
 
+			// Override svg icons color.
+			$template .= '
+				.tribe-common .tribe-common-c-svgicon {
+					color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:145]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 693079d974be1c3bbb92cedbb7bee707</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 4]
```diff
@@ -139,6 +139,19 @@ public function get_css_template( $template ) {
...
+				}
+			';
+
+			$template .= '
+				.tribe-common .tribe-events-virtual-virtual-event__icon-svg {
+					color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:151]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 33ad740031da95accacaa90b1d85f75f</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 2 warning(s)

File: src/Tribe/Customizer/Global_Elements.php [hunk 5]
```diff
@@ -10,7 +10,7 @@
  *
  * @package The Events Calendar
  * @subpackage Customizer
- * @since 4.4
+ * @since TBD
```
Comment (paulmskim) [inline src/Tribe/Customizer/Global_Elements.php:13]:
```suggestion
 * @since 4.4
```

**Review by paulmskim — APPROVED** [review]

**Pull request merged**